shopgate/pwa
# Added support for Universal Links

# Description

Added support for Universal Links: opening the app from an universal link + tracking custom events and campaign urls.

## Type of change

Please add an "x" into the option that is relevant:


## How to test it

Open the app from universal link, check if it opens correct page and sends tracking events.
